fe/fe-core/src/test/java/com/starrocks/sql/plan/PlanFragmentTest.java

```diff
@@ -2248,7 +2247,8 @@ public void testSetOperation() throws Exception {
                 + "   union all\n"
                 + "   (select * from db1.tbl6 where k1='b' and k4=5)\n"
                 + "   order by 3 limit 3)";
-        starRocksAssert.query(sql2).explainContains("UNION", 5);
+        System.out.println(starRocksAssert.query(sql2).explainQuery());
```

Comment:
It seems a debug log, can it be removed?

Reply:
sorry, my fault